stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/cookbook/ui/AttachmentGallery.kt

```diff
@@ -0,0 +1,41 @@
+package io.getstream.chat.docs.cookbook.ui
+
+import androidx.fragment.app.Fragment
+import com.getstream.sdk.chat.ChatUI
+import io.getstream.chat.android.ui.gallery.AttachmentGalleryDestination
+
+/**
+ * @see <a href="https://github.com/GetStream/stream-chat-android/wiki/UI-Cookbook#attachment-gallery">Attachment Gallery</a>
```

Comment:
I don't see anything in this link, only the option to create a new page called `UI Cookbook` is it supposed to be like that?

Reply:
I moved these pages because I was stupid, I'll move them back 😅